src/stores/notification.ts

```diff
@@ -7,7 +7,7 @@ import {
 import { S_STORAGE_AVAILABLE, GlobalStore } from '@/setup/globals';
```

Comment:
todo: please update this file according to the project from beginning of last year to get rid of the unnecessary types.

Reply:
I've updated `globals.ts `and `notification.ts` in a separate pull request, which is mainly focusing on adding axios response types. 

Let's only have the changes on [PR:347](https://github.com/valantic/vue-template/pull/347.) please.

Reply:
I did my best to only make necessary changes in this pull request regarding "notifications".